Documentation/WritingReST/ContentMenu.rst

```diff
@@ -0,0 +1,75 @@
+.. include:: /Includes.rst.txt
+.. highlight:: rst
+.. index::
+   Menus; Content
+.. _content-menu:
+
+============
+Content menu
+============
+
+You can display the titles of the current page as content menu.
+
+The simplest content menu can be created like this::
+
+   .. contents::
+      :local:
+
+Showing only local content menu of all headlines, excluding the page header:
+
+.. contents::
+   :local:
+
+You can also limit the levels and or give the contents menu a title::
+
+   .. contents:: Table of Contents
+      :local:
+      :depth: 1
+
+To limit the header to a certain depth:
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+You can also
```

Comment:
The sentence is not ended, is this intended?

Reply:
Thats true, found out the other options have no effect anymore, forgot to delete the stub sentence